src/node.cc

```diff
@@ -1110,7 +1110,7 @@ int Start(int argc, char** argv) {
       v8::StartupData* blob = NodeMainInstance::GetEmbeddedSnapshotBlob();
       if (blob != nullptr) {
         params.snapshot_blob = blob;
-        indexes = NodeMainInstance::GetIsolateDataIndexes();
+        indexes = NodeMainInstance::GetIsolateDataIndices();
```

Comment:
`indexes` too (can't suggest because there are references too far above and below this line)